apps/wing-console/console/server/src/router/app.ts

```diff
@@ -53,6 +53,13 @@ export const createAppRouter = () => {
         config: ctx.layoutConfig,
       };
     }),
+    "app.restart": createProcedure.mutation(async ({ ctx }) => {
```

Comment:
I can't see this log when I am playing around with this...

Reply:
it's a verbose message
<img width="399" alt="image" src="https://github.com/winglang/wing/assets/5547636/e6cac30c-8c2c-4cf9-a7d7-2a39680b1619">